app/Http/Controllers/PolicyController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Policy;
+use Illuminate\Http\Request;
+
+class PolicyController extends Controller
+{
+    public function getPolicies()
+    {
+        $policies = Policy::all();
+
+        return response()->json($policies);
+    }
+
+    public function editPolicyPeriod(Request $request, $id)
+    {
+        $policy = Policy::findOrFail($id);
+        $period = $request->input('period');
+
+        if ($period <= 0) {
```

Comment:
@radosevic-dr Rather use a validator for these kinds of checks

Reply:
Check updated